### PR TITLE
SB-24527  feat :Observation Without Rubric

### PR DESCRIPTION
**Pull request by vishwanath1004**
SB-24527  feat :Observation Without Rubric

### CODE REVIEW

File: src/app/client/src/app/modules/observation/components/edit-submission/edit-submission.component.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h4> {{resourceService.frmelmnts?.lbl?.instanceName}} </h4>
         </button>
         <div class="w-10"></div>
         <button type="button" class="ui sb-btn sb-btn-sm sb-btn-primary text-uppercase flex-basis-1" type="submit"
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/observation/components/edit-submission/edit-submission.component.html:20]: need required validation here

File: src/app/client/src/app/modules/observation/components/edit-submission/edit-submission.component.ts [hunk 2]
```diff
@@ -11,7 +11,6 @@ export class EditSubmissionComponent implements OnInit {
     @ViewChild('modal', { static: false }) modal;
     @Input() submission;
     @Output() onAction = new EventEmitter<any>();
-    selectedListCount=0;
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/observation/components/edit-submission/edit-submission.component.ts:14]: why is this deleted?

File: src/app/client/src/app/modules/observation/components/observation-details/observation-details.component.html [hunk 3]
```diff
@@ -60,15 +60,61 @@ <h4 class="font-weight-bold">{{resourceService.frmelmnts?.lbl?.observationDetail
...
-              (onAction)="actionOnSubmission($event)"> </submission>
+            <div *ngFor="let submission of submissions">
+              <!-- Observation With Rubric -->
+              <submission [submission]="submission" [allowMultipleAssessemts]="entities?.allowMultipleAssessemts"
+                *ngIf="submission?.evidencesStatus?.length > 1  ||  submission.isRubricDriven"
+                (selectedSubmission)="redirectToQuestions($event)" (onAction)="actionOnSubmission($event)">
+              </submission>
+
+              <!-- Observation Without Rubric -->
+              <div content class="pt-0 pb-8" *ngIf="submission?.evidencesStatus?.length == 1">
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/observation/components/observation-details/observation-details.component.html:78]: This should be a separate components

File: src/app/client/src/app/modules/observation/components/observation-details/observation-details.component.html [hunk 4]
```diff
@@ -60,15 +60,61 @@ <h4 class="font-weight-bold">{{resourceService.frmelmnts?.lbl?.observationDetail
...
             </div>
             <div class="twelve wide column" *ngIf="submissions?.length === 0 && !showLoader">
               <app-no-result [data]="noResultMessage"></app-no-result>
             </div>
-            <submission [submissions]="submissions" [allowMultipleAssessemts]="entities?.allowMultipleAssessemts"
-              *ngIf="submissions?.length && !showLoader" (selectedSubmission)="redirectToQuestions($event)"
-              (onAction)="actionOnSubmission($event)"> </submission>
+            <div *ngFor="let submission of submissions">
+              <!-- Observation With Rubric -->
+              <submission [submission]="submission" [allowMultipleAssessemts]="entities?.allowMultipleAssessemts"
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/observation/components/observation-details/observation-details.component.html:72]: @rejneesh1 Please check if these conditions are correct?

File: src/app/client/src/app/modules/observation/components/edit-submission/edit-submission.component.ts (hunk 2 above)
Comment (vishwanath1004) [reply src/app/client/src/app/modules/observation/components/edit-submission/edit-submission.component.ts:14]: Thats not using so i removed that.

File: src/app/client/src/app/modules/observation/components/observation-details/observation-details.component.html (hunk 4 above)
Comment (rejneesh1) [reply src/app/client/src/app/modules/observation/components/observation-details/observation-details.component.html:72]: @vishwanath1004 
successData.assessment.evidences.length > 1 || (submission.criteriaLevelReport && submission.isRubricDriven)

File: src/app/client/src/app/modules/observation/components/add-entity/add-entity.component.ts [hunk 5]
```diff
@@ -104,7 +104,7 @@ export class AddEntityComponent implements OnInit {
                 this.count = resp.count;
             }
         }, error => {
-            this.showLoader = false;
+            // this.showLoaderBox = false;
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/observation/components/add-entity/add-entity.component.ts:107]: rm if not required

File: src/app/client/src/app/modules/observation/components/add-entity/add-entity.component.ts [hunk 6]
```diff
@@ -132,9 +133,13 @@ export class AddEntityComponent implements OnInit {
         };
         this.observationService.post(paramOptions).subscribe(data => {
             this.closeModal();
-            this.showLoader = false;
         }, error => {
-            this.showLoader = false;
         })
     }
+
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/observation/components/add-entity/add-entity.component.ts:139]: rm if not required

File: src/app/client/src/app/modules/observation/components/observation-details/observation-details.component.ts [hunk 7]
```diff
@@ -41,21 +53,22 @@ export class ObservationDetailsComponent implements OnInit {
...
+      this.programName = data.programName
     })
   }
 
   ngOnInit() {
     this.getProfileData();
   }
   getProfileData() {
-    this.showLoader = true;
+    // this.showLoader = true;
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/observation/components/observation-details/observation-details.component.ts:65]: rm all commneted codes
